pkg/operator/controller/ingress/deployment.go

```diff
@@ -1196,6 +1224,14 @@ func serializeCaptureHeaders(captureHeaders []operatorv1.IngressControllerCaptur
 	return strings.Join(headerSpecs, ",")
 }
 
+func getHTTPHeadersListForSet(setHeaders operatorv1.IngressControllerHTTPHeader) string {
+	return fmt.Sprintf("%s:%s:%s", b64.StdEncoding.EncodeToString([]byte(setHeaders.Name)), b64.StdEncoding.EncodeToString([]byte(setHeaders.Action.Set.Value)), b64.StdEncoding.EncodeToString([]byte(setHeaders.Action.Type)))
+}
+
+func getHTTPHeadersForDelete(deleteHeaders operatorv1.IngressControllerHTTPHeader) string {
```

Comment:
To keep it consistent with `getHTTPHeadersListForSet`.  Alternatively, you could change `getHTTPHeadersListForSet` to match `getHTTPHeadersForDelete`, and become `getHTTPHeadersForSet`
```suggestion
func getHTTPHeadersListForDelete(deleteHeaders operatorv1.IngressControllerHTTPHeader) string {
```

Reply:
done